scripts/convert_bed.py

```diff
@@ -0,0 +1,36 @@
+import pybedtools
+
+
+def convert_chr_style(
+    interval: pybedtools.Interval, chr_style: str
+) -> pybedtools.Interval:
+    if chr_style == '' and 'chr' in interval.chrom:
```

Comment:
A bit nitpicky, but it would be nice to add a docstring to clarify the purpose of the function. It's pretty clear from the code in thise case, but it's a good practice anyway.